cli/pom.xml

```diff
@@ -29,6 +29,11 @@
     <description>Hono Command line Interface</description>
 
     <dependencies>
+        <dependency>
+            <groupId>commons-cli</groupId>
+            <artifactId>commons-cli</artifactId>
```

Comment:
I do not really understand what we need this for. Spring Boot supports binding command line params to fields or beans out-of-the-box ...